spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/intermediate/IRConverter.scala

```diff
@@ -91,8 +91,11 @@ private[bigdl] class IRConverter[T: ClassTag](IRgraph: IRGraph[T])(implicit ev:
     // add output node for graph
     val realOutputs = outputs.zipWithIndex.map {
       case (model: Node[Module[Float]], index: Int) =>
-        val node = new Node[Module[Float]](Output(IRgraph.outputFormats(index)))
-        model.add(node)
+        val node = if (model.element.isInstanceOf[BlasWrapper]) {
+          model
+        } else {
+          model.add(new Node[Module[Float]](Output(IRgraph.outputFormats(index))))
+        }
```

Comment:
why we skip all blas nodes here ?

Reply:
there is no need to add reorder in blas wrapper layer.